src/fidesops/util/async_util.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+from asyncio import AbstractEventLoop
+from typing import TypeVar, Callable, Any, Awaitable, Optional
+import logging
+
+logger = logging.getLogger(__name__)
+T = TypeVar("T")
+
+
+def _loop() -> AbstractEventLoop:
+    """Return the event loop"""
+    asyncio.set_event_loop(asyncio.SelectorEventLoop())
+    return asyncio.get_event_loop()
+
+
+def run_async(task: Callable[[Any], T], *args: Any) -> Awaitable[T]:
+    """Run a callable async"""
+    if not callable(task):
+        raise TypeError("Task must be a callable")
+
+    return _loop().run_in_executor(None, task, *args)
```

Comment:
Can you explain why running the privacy request in a separate thread is a prerequisite for dask async?